src/core/utils.js

```diff
@@ -822,3 +822,13 @@ export class InsensitiveStringSet extends Set {
         );
   }
 }
+
+export function makeSafeCopy(node) {
+  const clone = node.cloneNode(true);
+  clone.querySelectorAll("[id]").forEach(elem => elem.removeAttribute("id"));
+  clone.querySelectorAll("dfn").forEach(dfn => renameElement(dfn, "span"));
+  // TODO: make a tree walker and remove all the comment nodes, not just first level.
+  return [...clone.childNodes].filter(
```

Comment:
Filtering is done later by exporter, why should be done here?

Reply:
I was worried that we might pick up these nodes again and pass them to HyperHTML in a different/future module. I don’t know exactly what HyperHTML does with these nodes, and if it might get confused.

Reply:
Hmm. Maybe we could get a general wrapper to filter comments out.

Reply:
Yeah, i looked at pulling the one out you created from the exporter. Should be easy to do.